frontend/src/pages/org/crawl-detail.ts

```diff
@@ -797,8 +794,19 @@ ${this.crawl?.description}
                       </a>
                     </div>
                     <div
-                      class="whitespace-nowrap text-sm font-mono text-neutral-400"
+                      class="whitespace-nowrap text-sm font-mono text-neutral-400 flex items-end"
                     >
+                      ${when(
+                        file.numReplicas > 0,
+                        () => html` <sl-tooltip
+                          content=${msg("File replicated")}
+                        >
+                          <sl-icon
+                            name="cloud-check"
+                            class="h-4 pr-2 shrink-0 text-success"
```

Comment:
I think this should fix the tooltip alignment issue — `pr` is padding-right, which is inside the box, whereas `mr` is margin-right, which is outside the box. The tooltip is seeing two extra pixels of width inside the box it's positioning itself relative to, and instead we just want to push the whole box over 2px without making it wider.
```suggestion
                            class="h-4 mr-2 shrink-0 text-success"
```